server/tests/src/test/java/org/infinispan/server/security/AuthenticationImplicitIT.java

```diff
@@ -30,67 +32,86 @@
 
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
- * @since 10.1
+ * @since 11.0
  **/
 
 @RunWith(Parameterized.class)
 @Category(Security.class)
-public class AuthenticationKerberosSpnegoIT {
+public class AuthenticationImplicitIT {
    @ClassRule
    public static InfinispanServerRule SERVERS =
-         InfinispanServerRuleBuilder.config("configuration/AuthenticationKerberosTest.xml")
-                                    .numServers(1)
-                                    .property("java.security.krb5.conf", "${infinispan.server.config.path}/krb5.conf")
+         InfinispanServerRuleBuilder.config("configuration/AuthenticationServerImplicitTest.xml")
                                     .build();
 
    @ClassRule
-   public static LdapServerRule LDAP = new LdapServerRule(SERVERS, "ldif/infinispan-kerberos.ldif", true);
+   public static LdapServerRule LDAP = new LdapServerRule(SERVERS);
 
    @Rule
    public InfinispanServerTestMethodRule SERVER_TEST = new InfinispanServerTestMethodRule(SERVERS);
 
-   private final Protocol protocol;
    private final String mechanism;
-
-   private static String oldKrb5Conf;
+   private final String protocol;
 
    @Parameterized.Parameters(name = "{1}({0})")
    public static Collection<Object[]> data() {
-      List<Object[]> params = new ArrayList<>(HTTP_KERBEROS_MECHS.size() * HTTP_PROTOCOLS.size());
+      List<Object[]> params = new ArrayList<>();
+      for(Object[] mech : Common.SASL_MECHS) {
+         params.add(new Object[]{"Hot Rod", mech[0]});
+      }
       for (Protocol protocol : HTTP_PROTOCOLS) {
-         for (Object[] mech : HTTP_KERBEROS_MECHS) {
-            params.add(new Object[]{protocol, mech[0]});
+         for (Object[] mech : HTTP_MECHS) {
+            params.add(new Object[]{protocol.name(), mech[0]});
          }
       }
       return params;
    }
 
-   public AuthenticationKerberosSpnegoIT(Protocol protocol, String mechanism) {
+   public AuthenticationImplicitIT(String protocol, String mechanism) {
       this.protocol = protocol;
       this.mechanism = mechanism;
    }
 
-   @BeforeClass
-   public static void setKrb5Conf() {
-      oldKrb5Conf = System.setProperty("java.security.krb5.conf", AuthenticationKerberosSpnegoIT.class.getClassLoader().getResource("configuration/krb5.conf").getPath());
+   @Test
+   public void testProtocol() {
+      if ("Hot Rod".equals(protocol)) {
+         testHotRod();
+      } else {
+         testRest(Protocol.valueOf(protocol));
+      }
    }
 
-   @AfterClass
-   public static void restoreKrb5Conf() {
-      if (oldKrb5Conf != null) {
-         System.setProperty("java.security.krb5.conf", oldKrb5Conf);
+   public void testHotRod() {
+      ConfigurationBuilder builder = new ConfigurationBuilder();
+      if (!mechanism.isEmpty()) {
+         builder.security().authentication()
+               .saslMechanism(mechanism)
+               .serverName("infinispan")
```

Comment:
i the server name mandatory now ?

Reply:
Actually, it already defaults to `infinispan`